README.md

```diff
@@ -110,6 +110,36 @@ The image will be rendered inside the displayed dataframe and next to the "gener
 
 (*Note: you also need an `OPENAI_API_KEY` environment variable to use `gpt-4-vision-preview`*)
 
+## Using input data in prompts
+
+The user/system prompt textboxes support [Jinja](https://jinja.palletsprojects.com/) templates. 
+Given a column named `text` in your uploaded CSV data, you can use values from this column by 
+writing the simple `{{text}}` template in your prompt.
+
+If the values in your column represent more complex objects such as Python dictionaries or lists,
+you can still work with them but make sure that these values are valid string representations and
```

Comment:
Not valid JSON representations?

Reply:
`ast.literal_eval` (in `fromAstString`) will happily parse JSON-invalid stuff, e.g. `{'A': 'B'}`.